docs/kyma-release-notes/2023-10-02-release-notes-2.19.md

```diff
@@ -0,0 +1,36 @@
+
+## Application Connectivity
+
+Additional logging options have been added to Central Application Gateway.
+https://github.com/kyma-project/kyma/#debugging
+
+## Observability
+
+Early warning: The Monitoring component will be removed with Kyma 2.20, as announced in this [blog post](https://blogs.sap.com/2023/09/07/removal-of-prometheus-grafana-based-monitoring-in-sap-btp-kyma-runtime/)
+
+Update of components:
+ - kube-state-metrics 2.10.0 (https://github.com/kyma-project/kyma/pull/18135)
+ - oauth2-proxy 7.5.1 (https://github.com/kyma-project/kyma/pull/18222)
+
+## Telemetry
+
+The Telemetry component has been [replaced by the module](https://github.com/kyma-project/kyma/issues/16301) and won't be part of the Kyma release notes anymore. Instead, learn about upcoming releases of the Telemetry module on https://github.com/kyma-project/telemetry-manager/releases.
+	
+## Service Mesh
+
+With Kyma 2.19, we have updated the Istio component to version 1.1.0. It contains the following changes:
+ - Istio updated to version 1.19.0
+ - Enabled Horizontal Pod Autoscaling capability for a smaller cluster installation
+ - Additional Kyma resources and configuration, such as PeerAuthentication configuring service-mesh traffic to only allow mTLS
```

Comment:
```suggestion
 - [Additional Kyma resources and configuration](https://github.com/kyma-project/istio/issues/334), such as Istio Grafane dashboards or PeerAuthentication configuring service-mesh traffic to only allow mTLS
```
Let's combine those two points because these tasks were part of the same issue.

Reply:
it shall be Grafana instead of Grafane.